megazords/fenix/android/build.gradle

```diff
@@ -106,10 +106,13 @@ afterEvaluate {
 }
 
 apply from: "$rootDir/publish.gradle"
+// If using jnaForTestConfiguration or variantWithoutLib,
+// please also update the corresponding .buildconfig-android.yml
+// `publishedArtifacts` property.
 ext.configurePublish(
-        'org.mozilla.appservices',
-        'fenix-megazord',
-        'Megazord of [fxaclient, places, push]',
+        rootProject.ext.library.groupId,
```

Comment:
These are getting set from the `.yml`, right?  If this is uniform, why not fish these values in `configurePublish` rather than passing them around?